src/test/java/easytests/core/mappers/TopicsMapperTest.java

```diff
@@ -15,84 +15,93 @@
  * @author malinink
  */
 public class TopicsMapperTest extends AbstractMapperTest {
+
+    private TopicsSupport topicsSupport = new TopicsSupport();
+
     @Autowired
     private TopicsMapper topicsMapper;
 
     @Test
     public void testFindAll() throws Exception {
-        final List<TopicEntity> topicsEntities = this.topicsMapper.findAll();
-        Assert.assertEquals((Integer) 3, (Integer) topicsEntities.size());
+        final List<TopicEntity> topicEntities = this.topicsMapper.findAll();
+
+        Assert.assertEquals(3, topicEntities.size());
+
+        Integer index = 0;
+        for(TopicEntity eachTopicEntity: topicEntities) {
```

Comment:
Why not topicEntity ?  `eachTopicEntity`

Reply:
@malinink 
Well, I don't know. Actually, I supposed it's a better name to "for each" type cycle, and so I named it just like this.